src/CrawlAbstractLvl/private/abslvlworld.h

```diff
@@ -0,0 +1,32 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the GPLv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#ifndef WORLD_H
```

Comment:
please change this define name to the name of the class .
example : 
ABSLEVELWORLD_H